CollabApp/CollabApp.mvc/Views/Post/Index.cshtml

```diff
@@ -13,10 +13,6 @@
     <h1>Posts</h1>
     <div class="container">
         <form asp-asp-controller="Post" asp-action="Index" method="post">
-            <div class="row">
-                <div class="col-1">Author</div>
```

Comment:
dabar posto autorius niekur nėra set'inamas.
![image](https://github.com/niasenago/PSI-project/assets/118674771/de929894-4cfb-416e-a5ed-0885d0d22b69)

Reply:
It should be fixed now.